shared/oae/api/oae.api.admin.js

```diff
@@ -292,6 +292,34 @@ define(['exports', 'jquery', 'underscore'], function(exports, $, _) {
     };
 
     /**
+     * Delete a user
```

Comment:
Should we add a note about the user being soft deleted and the ability to restore the user?